### PR TITLE
Adds Jerry the Baby Reaver and Maps It in as a OM Pet

### PR DESCRIPTION
**Pull request by Generalcamo**
Adds Jerry, the baby reaver. The winner of a poll conducted on the #scc_bulletin on the relay.

Also adds sprites for the eel, the runner up.

### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| icons/mob/npc/pets.dmi | Yonnimer (Aurorastation) | CC-BY-SA |

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/reaver.dm [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	flying = TRUE
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+	density = FALSE
+	mob_size = MOB_SMALL
+	speed = 4
+	maxHealth = 50
+	health = 50
+
+	response_help = "pets the"
```
Comment (Greenjoe12345) [inline code/modules/mob/living/simple_animal/friendly/reaver.dm:22]: Wouldn't this lead to "pets the the baby reaver"?

Comment (Generalcamo) [reply code/modules/mob/living/simple_animal/friendly/reaver.dm:22]: This is the exact phrase used for space carp, including the enemy reaver. I do not believe this is proper/improper aware

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
add a 1 tile cage for Jerry. refer to the other pets' cages for inspiration.

**Comment by DreamySkrell** [comment]
> add a 1 tile cage for Jerry. refer to the other pets' cages for inspiration.

I don't think a 1 tile cage is necessary. They kinda look bad now with 3/4. I have intentionally removed Ian's cage during the 3/4 remaps (third deck was my assignment).

Currently half of the pets do not have cages (Ian, Chauncey, Crusher).
And those that do have cages (Ginny, Ives, Columbo) are "docile" variants of dangerous beasts (space carp, hivebot... and, uh, dog).

**Comment by DreamySkrell** [comment]
I think a baby reaver would be an extra lame pet for cargo.
It's just a meme cause it kills miners often. Reavers have have zero lore - even hivebots (Ives) have actual lore, as little as they do.
Personally, I don't think we need more pets, and definitely not memes like this.

**Comment by NonQueueingMatt** [comment]
Reavers are a naturally hostile animal, we don't need people pretending that they're cute and misunderstood

**Pull request closed without merging**